src/main/webapp/WEB-INF/static/js/fieldMap.js

```diff
@@ -805,96 +801,16 @@ function doEnterFieldDetailsPageLoad() {
 		prevFieldId = $('#' + getJquerySafeId('userFieldmap.fieldId')).val(),
 		prevBlockId = $('#' + getJquerySafeId('userFieldmap.blockId')).val();
 
-	$('#' + getJquerySafeId('userFieldmap.fieldLocationId')).val('');
+	$('#' + getJquerySafeId('userFieldmap.fieldLocationId')).val(defaultLocationId);
+	$('#' + getJquerySafeId('userFieldmap.locationName')).val(defaultLocationName);
+	$('#' + getJquerySafeId('userFieldmap.numberOfRowsPerPlot')).val(defaultRowsPerPlot);
+
 	programLocationUrl = $('#programLocationUrl').val();
 	setSelectedTrialsAsDraggable();
 	calculateTotalPlots();
 
-	recreateLocationCombo();
-
-	initializeFieldSelect2({}, [], true);
-	initializeBlockSelect2({}, [], true);
-
-	showBlockDetails(true, null);
-
-	// remove any other listeners for the location update
-	$(document).off('location-update');
-	$(document).on('location-update', recreateLocationCombo);
-
-	$('#showFavoriteLocation, #showAllLocationRadio, #showBreedingLocationOnlyRadio').on('change', function() {
-		showCorrectFieldLocationCombo();
-		initializeFieldSelect2({}, [], true);
-		initializeBlockSelect2({}, [], true);
-		showBlockDetails(true, null);
-	});
-
-	var numRowPerPlot = $('#' + getJquerySafeId('userFieldmap.numberOfRowsPerPlot')).val();
-	$('#' + getJquerySafeId('userFieldmap.numberOfRowsPerPlot')).val(defaultRowsPerPlot);
-
-	if (prevFieldId != '') {
-		var favLocationChkElem =  $('#showFavoriteLocation');
-		var isChecked = favLocationChkElem.is(':checked');
-
-		$('#' + getJquerySafeId('userFieldmap.numberOfRowsPerPlot')).val(numRowPerPlot);
-
-		if (isChecked) {
-
-			if ($("#showBreedingLocationOnlyRadio").is(':checked')) {
-				$('#s2id_fieldLocationIdBreedingFavorites').show();
-				$('#s2id_fieldLocationIdFavorite').hide();
-				$('#s2id_fieldLocationIdAll').hide();
-				$('#s2id_fieldLocationIdBreeding').hide();
-				setCorrectValueToFieldCombo (locationSuggestionsBreedingFav_obj, prevFieldLocationId, 'fieldLocationIdBreedingFavorite')
-			} else {
-				$('#s2id_fieldLocationIdBreedingFavorites').hide();
-				$('#s2id_fieldLocationIdFavorite').show();
-				$('#s2id_fieldLocationIdAll').hide();
-				$('#s2id_fieldLocationIdBreeding').hide();
-				setCorrectValueToFieldCombo (locationSuggestionsFav_obj, prevFieldLocationId, 'fieldLocationIdFavorite');
-			};
-
-		} else {
+	showBlockDetails(false, null);
```

Comment:
Same here, should we rename `showBlockDetails`?

Reply:
Hi @darla-leafnode , I removed the unused scripts